pkgs/applications/misc/stacktile/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromSourcehut
+, wayland
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "stacktile";
+  version = "1.0.0";
+
+  src = fetchFromSourcehut {
+    owner = "~leon_plickat";
+    repo = pname;
+    fetchSubmodules = true;
+    rev = "v${version}";
+    sha256 = "sha256-IOFxgYMjh92jx2CPfBRZDL/1ucgfHtUyAL5rS2EG+Gc=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    wayland
+  ];
+
+  makeFlags = [
+    "DESTDIR=${placeholder "out"}"
+  ];
+
+  meta = with lib; {
+    homepage = "https://sr.ht/~leon_plickat/stacktile";
+    description = "A layout generator for river Wayland compositor.";
```

Comment:
According to https://nixos.org/manual/nixpkgs/unstable/#var-meta-description descriptions shouldn't include a period in the end.